compiler/rustc_mir_build/src/builder/scope.rs

```diff
@@ -571,23 +573,47 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
     /// branch to.
     pub(crate) fn in_const_continuable_scope<F>(
         &mut self,
-        loop_head: BasicBlock,
         match_arms: SwitchTargets,
         state_place: Place<'tcx>,
+        span: Span,
         f: F,
     ) -> BlockAnd<()>
     where
         F: FnOnce(&mut Builder<'a, 'tcx>) -> Option<BlockAnd<()>>,
```

Comment:
I believe we can make this return `()`, the `Some` variant looks like it's only used for `break`s from a block, not when it's used in loops. That would simplify the logic below

Reply:
Done